runalltests.sh

```diff
@@ -1,2 +1,7 @@
 #!/usr/bin/env sh
-matlab -nosplash -nodesktop -r "batchTesting(0);exit"
+if [ -d "data" ]; then
+  matlab -nosplash -nodesktop -r "batchTesting(0);exit"
+else
+  echo "Cannot run tests - please obtain all test files first with the get_testfiles.py script."
```

Comment:
can't this get_testfiles python script be run within the bashscript?

Reply:
Yeah, I thought of that but it would be confusing and slow. The intent of the script is to run the tests not obtain them. 

Also, the download script can take a while and this can confuse the user or create problems if the python dependencies are not met for example.